core/bind/core_bind.cpp

```diff
@@ -520,6 +520,10 @@ Error _OS::shell_open(String p_uri) {
 	return OS::get_singleton()->shell_open(p_uri);
 };
 
+String _OS::read_string_from_stdin(bool p_block) {
+	return OS::get_singleton()->get_stdin_string(true);
```

Comment:
```suggestion
	return OS::get_singleton()->get_stdin_string(p_block);
```

But probably the argument should be removed at all (from both CoreBind and OS), it's not useful, setting it to `false` always return empty string.

Reply:
The argument was likely left for forward compatibility (in case non-blocking stdin reading is implemented in `3.x`), but it's not likely to happen in `3.x`.

Reply:
I would also remove the unused parameter from the OS method, not just the binding.